client/src/shared/App.tsx

```diff
@@ -47,8 +47,9 @@ export default class App extends React.Component {
         <Router>
           <React.Fragment>
             <Switch>
-              <Route exact path="/" component={Login}/>
-              <PrivateRoute component={TestSection} exact path="/test" isLoggedIn={true} />
+              <Route exact path="/" component={Login} />
+              <Route exact path="/educator-dashboard" component={EducatorDashboard} />
+              <Route path="/event-page" component={EventPage} />
```

Comment:
by exclusively using state to dictate how this page renders, we run into the problem if the user navigates via URL to `/event-page` -> the app crashes. 

this also doesn't allow users to bookmark individual events (a lot of old people do this lol)

at the very least we'll have to add a redirect to the top of this page if the state isn't passed. ideally, we should redesign the loading of this page so that the event id is part of the URL/route (we'll talk about this to gauge the effort).

Reply:
I added in a name field in the routing so now we can get over bookmarking and the page now renders `/event/:name`. I don't believe we pull in event IDs but we could.

Reply:
individual event page in now `/events/:name` and all events are now in `/events/`. Resolved in 
79a2fd1